setup.cfg

```diff
@@ -34,6 +34,7 @@ install_requires =
     numpy>1.17.3, <2.0.0
     td-client>=1.1.0
     pytz>=2018.5
+    tqdm<=4.67.0
```

Comment:
Is there any reason to set a strict upper boundary for tqdm? The latest version is 4.67.1 https://pypi.org/project/tqdm/4.67.1/

It could cause future installation issues by conflicting with dependencies out of pytd when pytd release is delayed or becoming inactive.

Reply:
No particular reason, I'll downgrade it to >=4.60.0 (2022) or something along those lines